config/locales/en.yml

```diff
@@ -50,8 +50,8 @@ en:
     errors:
       models:
         bulk_upload/row_parser: &bulk_upload__row_parser__base
-          inclusion: Enter a valid value for %{question}
-          spreadsheet_dupe: This is a duplicate of a log in your file
+          inclusion: "Enter a valid value for %{question}"
```

Comment:
This stays without a full stop because the question itself will have one/a question mark.